.github/workflows/build_sign_release.yaml

```diff
@@ -32,42 +32,60 @@ jobs:
     permissions:
       id-token: write
       contents: write
+    defaults:
+      run:
+        shell: powershell
     # This condition prevents duplicate runs.
     if: github.event_name != 'pull_request' || github.event.pull_request.head.repo.full_name != github.event.pull_request.base.repo.full_name
     steps:
       - name: Checkout
         uses: actions/checkout@v4
         with:
           path: repo
-      - name: OIDC Login to Azure Public Cloud with AzPowershell (enableAzPSSession true)
+      - name: Install Azure Signing Tool
+        run: |
+          dotnet --version
+          dotnet tool install --global AzureSignTool --version 5.0.0
+      # OIDC Login to Azure Public Cloud with AzPowershell (enableAzPSSession true)
+      - name: Login to Azure
         uses: azure/login@v2
         with:
           client-id: ${{ secrets.AZURE_CLIENT_ID }}
           tenant-id: ${{ secrets.AZURE_TENANT_ID }}
           subscription-id: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
           enable-AzPSSession: true
-      - name: Sign Module
-        uses: azure/powershell@v1
+      - name: Get Key Vault info
+        id: key-vault-info
         env:
           KEY_VAULT_INFO: ${{ secrets.SCUBA_KEY_VAULT_PROD}}
-        with:
-          inlineScript: |
-            dotnet --version
-            dotnet tool install --global AzureSignTool --version 4.0.1
-            $KeyVaultInfo = ${env:KEY_VAULT_INFO} | ConvertFrom-Json
-            . repo/utils/DeployUtils.ps1
-            # Remove non-release files
-            Remove-Item -Recurse -Force repo -Include .git*
-            SignScubaGearModule `
-              -AzureKeyVaultUrl $($KeyVaultInfo.KeyVault.URL) `
-              -CertificateName $($KeyVaultInfo.KeyVault.CertificateName) `
-              -ModulePath 'repo'
-            Move-Item  -Path repo -Destination "ScubaGear-${env:RELEASE_VERSION}" -Force
-            Compress-Archive -Path "ScubaGear-${env:RELEASE_VERSION}" -DestinationPath "ScubaGear-${env:RELEASE_VERSION}.zip"
-          azPSVersion: "latest"
-      - name: release
+        run: |
```

Comment:
For this run from line 61-64, shouldn't there a name associated with this run or is it all part of getting the get key vault info?

Reply:
There are 2 jobs here, starting with lines 57 and 65, and both have their own names.  Probably hard to see this from just looking at the change page on GitHub, as it often chops things into little pieces.